src/user-guide/running-suites.rst

```diff
@@ -644,7 +644,6 @@ These can be configured using the settings:
 
 - :cylc:conf:`[mail]to` (list of email addresses)
 - :cylc:conf:`[mail]from`
-- :cylc:conf:`[mail]smtp`
```

Comment:
The SMTP server can still be set, but only in global config. Maybe it's worth linking to that?

Reply:
I don't think so - I think that we'd rather most users don't tinker with the global config.

Reply:
Ok fair enough